src/main/java/org/spongepowered/asm/util/Bytecode.java

```diff
@@ -503,7 +503,13 @@ public static String getOpcodeName(AbstractInsnNode node) {
      *      the {@link Opcodes} class have the same value as opcodes
      */
     public static String getOpcodeName(int opcode) {
-        return Bytecode.getOpcodeName(opcode, "UNINITIALIZED_THIS", 1);
+        if (opcode < 0) {
+            return "UNKNOWN";
+        }
+        if (opcode < OpcodeNames.INSNS.length) {
+            return OpcodeNames.INSNS[opcode];
+        }
+        return String.valueOf(opcode);
```

Comment:
Why not fall back to the old path to handle future opcodes?

Reply:
Switched to the asm utility which will presumably be updated as opcodes are added.